{{cookiecutter.github_repository}}/.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 sudo: false
 python:
-- {{ '3.5' if cookiecutter.python3.lower() == 'y' else '2.7' }}
+- {{ '3.5.1' if cookiecutter.python3.lower() == 'y' else '2.7.11' }}
```

Comment:
@jasonrfarkas have you verified against travis that this works, if not i'm happy to have it added include in another pull-request.